x-pack/plugins/security_solution/server/usage/detections/rules/transform_utils/get_rule_object_correlations.ts

```diff
@@ -55,6 +55,9 @@ export const getRuleObjectCorrelations = ({
       cases_count_total: casesRuleIds.get(ruleId) || 0,
       has_legacy_notification: hasLegacyNotification,
       has_notification: hasNotification,
+      has_legacy_investigation_field: attributes.params.investigationFields
```

Comment:
Won't be `has_legacy_investigation_field: Array.isArray(attributes.params.investigationFields),` enough here?